src/paperless/settings.py

```diff
@@ -762,8 +762,12 @@ def _parse_db_settings() -> dict:
 # django setting.
 CACHES = {
     "default": {
-        "BACKEND": "django.core.cache.backends.redis.RedisCache",
+        "BACKEND": os.environ.get(
+            "PAPERLESS_CACHE_BACKEND",
+            "django.core.cache.backends.redis.RedisCache",
+        ),
         "LOCATION": _CHANNELS_REDIS_URL,
+        "KEY_PREFIX": os.getenv("PAPERLESS_REDIS_PREFIX", ""),
```

Comment:
Intentionally undocumented?

Reply:
That setting is documented.  It's usage here is for the same thing, to enable sharing a broker with multiple instances, for anyone doing it

Reply:
Sorry, I meant the `PAPERLESS_CACHE_BACKEND`, but its definitely not a big deal, I was just curious

Reply:
Ah, that one is deliberate.  Otherwise testing would need a Redis server.  There's no reason a user should ever need to know or set that

Reply:
Cool, thanks